src/selector/isChildOf.js

```diff
@@ -0,0 +1,13 @@
+(function(factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['$'], factory);
+    } else {
+        factory(window.Zepto || window.jQuery);
+    }
+}(function($) {
+    $.extend($.fn, {
+        isChildOf: function(selector) {
+            return (selector) ? $(this).parents().filter(selector).length > 0 : false;
```

Comment:
@scalvert Correct me if I'm wrong, but when you pass in an empty selector to filter(), it seems that it won't filter anything. Then .parents() will return all parent elements, which results in returning true. 

The intent is to return false on "invalid inputs", such as an empty selector. I can't imagine anyone passing in an empty selector when calling "isChildOf" other than by mistake, which is why I want it to return false. What are your thoughts?

Also, why do we need the .filter() after the parents()? Can't we pass in the selector in the parents()?